src/main/java/domain/Users.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+import exception.ErrorMessage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Users {
+    private final List<User> users;
```

Comment:
```suggestion
    private final List<User> users = new ArrayList<>();
```

필드에 바로 초기화하면, 생성자에서 따로 초기화할 필요 없지 않을까요?

Reply:
다니가 말한 방식으로 하면 생성자에서 한 줄을 더 줄일 수 있겠네요!